00000.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------

```

Comment:
A bunch of these files are in the root of the repo. They obviously shouldn't be here